Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3
```

Comment:
What is this Dockerfile used for?

Reply:
This is the test runner for the "tests" service in the compose file. https://github.com/posit-dev/posit-sdk-py/blob/f8e38a79a7a33cae99f56c3431fe90170a281c2b/integration/docker-compose.yaml#L3